docs/index.md

```diff
@@ -13,25 +13,4 @@ manager: douge
 
 Visual Studio IntelliCode is a new tool that enhances software development using artificial intelligence. IntelliCode helps developers and teams code with confidence, focus code reviews, and find issues faster. IntelliCode extensions are available for [C# in Visual Studio](intellicode-visual-studio.md), and [Python in Visual Studio Code](intellicode-visual-studio-code.md). 
 
-<br />
-
-<ul class="panelContent cardsFTitle">
-    <li>
-        <a href="faq.md">
-        <div class="cardSize">
-            <div class="cardPadding">
-                <div class="card">
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <img src="https://docs.microsoft.com/media/logos/logo_vs-ide.svg" alt="Visual Studio logo">
```

Comment:
Instead of deleting this section, lets change line 26 to \<img height="60" width="60" src=…>

Reply:
That's a great solution!